clients/admin-ui/src/types/api/README.md

```diff
@@ -0,0 +1,27 @@
+# FidesCtl API Types
+
+**Do not manually edit these files.**
+
+The TypeScript files in this directory are auto-generated by OpenAPI. This ensures our frontend uses the same schema as the backend without needing to manually convert Python classes to TypeScript. Currently, we only generate model types.
+
+Use these types working with API requests & responses, for example:
+
+```ts
+import { DatasetField } from "~/types/api";
```

Comment:
The name "api" for this directory is simple, but I'd be open to other names like "fidesctl" or something.